Add option: ["camelCaseSvgKeywords"] to value-keyword-case #4884 (#5849)
stylelint/stylelint: lib/rules/value-keyword-case/README.md

  color: Foo(--camelCase);
}
```

### `camelCaseSvgKeywords: true | false` (default: `false`)

> If `true`, this rule will not check the camel case keyword.

Reviewer: ```suggestion
If `true`, this rule expects camelCase SVG keywords regardless of the primary option
```

For example with `true`:

> The following patterns are _not_ considered problems:

Reviewer 2: ```suggestion
The following pattern is _not_ considered a problem:
```

<!-- prettier-ignore -->
```css
a {

>   color: currentcolor;

Reviewer: ```suggestion
  color: currentColor;
```

}
```

> The following patterns are considered problems:

Reviewer 2: ```suggestion
The following pattern is considered a problem:
```

<!-- prettier-ignore -->
```css
a {

>   color: currentColor;

Reviewer: ```suggestion
  color: currentcolor;
```
